qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -629,3 +630,73 @@ def test_config_session_timeout(self):
         self.assert_session_count(1)
 
         self.mount_a.kill_cleanup()
+
+    def test_reconnect_after_blacklisted(self):
+        """
+        Test reconnect after blacklisted.
+        - writing to a fd that was opened before blacklist should return -EBADF
+        - reading/writing to a file with lost file locks should return -EIO
+        - readonly fd should continue to work
+        """
+
+        if not isinstance(self.mount_a, KernelMount):
+            self.skipTest("Not implemented in FUSE client yet")
+
+        self.mount_a.umount_wait()
+        self.mount_a.mount(mount_opts=['recover_session=clean'])
```

Comment:
This will probably need a try/except for a mount error for an invalid option, right? The distro kernels won't have this option yet and will all fail.